app/models/show.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Show < ApplicationRecord
```

Comment:
also you can use rails `enum` for the `show_type` it's linked in the issue. In the schema make the show_type: `t.integer :show_type, null: false` and let the the enum store the value as integer in database, it should show string when its displaying it